src/preferences/dialog/dlgprefinterface.cpp

```diff
@@ -191,14 +197,17 @@ void DlgPrefInterface::slotUpdateSchemes() {
         ComboBoxSchemeconf->setEnabled(false);
         ComboBoxSchemeconf->addItem(tr("This skin does not support color schemes", 0));
         ComboBoxSchemeconf->setCurrentIndex(0);
+        // clear m_colorScheme just in case it's not empty
+        m_colorScheme = QString();
     } else {
         ComboBoxSchemeconf->setEnabled(true);
-        QString selectedScheme = m_pConfig->getValueString(ConfigKey("[Config]", "Scheme"));
+        QString configScheme = m_pConfig->getValueString(ConfigKey("[Config]", "Scheme"));
         for (int i = 0; i < schlist.size(); i++) {
             ComboBoxSchemeconf->addItem(schlist[i]);
 
-            if (schlist[i] == selectedScheme) {
+            if (schlist[i] == configScheme) {
                 ComboBoxSchemeconf->setCurrentIndex(i);
+                m_colorScheme = configScheme;
```

Comment:
The only case where the preview is empty after start is when the skin and scheme from mixxx.cfg don't match.
The dropdown and GUi load the first scheme from skin.xml, though.
Is it worth catching that here with another if switch?